src/pages/CoreApps/CoreApps.js

```diff
@@ -0,0 +1,98 @@
+import { useDataQuery, useConfig } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { NoticeBox, CenteredContent, CircularLoader } from '@dhis2/ui'
+import React from 'react'
+import { AppsList } from '../../components/AppsList/AppsList'
+import { coreApps } from '../../core-apps'
+import { getLatestVersion } from '../../get-latest-version'
+import { semverGt } from '../../semver-gt'
+
+const query = {
+    coreApps: {
+        resource: 'apps',
+        params: {
+            bundled: true,
+        },
+    },
+    // TODO: Add ability to request certain app IDs to `/v2/apps` API and use
```

Comment:
Instead of fetching every App Hub app in order to see if any installed apps have pending updates, maybe we should have a `v2` API that given a list of app IDs returns data about those apps.

Reply:
Yeah, definitely agreed.  We could just request each app independently for now, and maybe implement a graphql API in the future to support batching.

Reply:
It would be nice to get a "count" of updates available (for each core apps and installed apps) though, to show a badge in the sidebar or similar.  That would also allow us to more easily implement an "update all" button